web/BUILD.yaml

```diff
@@ -11,7 +11,7 @@ steps:
 
   build:
     environment:
-      ELECTRICITYMAP_PUBLIC_TOKEN: ${BRICK_ELECTRICITYMAP_PUBLIC_TOKEN}
+      ELECTRICITYMAP_PUBLIC_TOKEN: ${BRICK_ELECTRICITYMAP_PUBLIC_TOKEN:-unknown}
```

Comment:
What does this `:-unknown` mean?

Reply:
it's a fallback value for when the env variable is not defined.
It's due to https://github.com/tmrowco/brick/issues/79